packages/validator/src/commands/convertCmd.ts

```diff
@@ -11,11 +11,7 @@ const ois2config = 'OIS2Config.json';
 
 if (process.env.npm_config_template) {
   console.log(
-    JSON.stringify(
-      convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template),
-      null,
-      '\t'
-    )
+    JSON.stringify(convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template), null, 2)
```

Comment:
If calling `JSON.stringify(something, null, 2)` is so common maybe we should have a helper method for that.

Reply:
Seems to be used only 3 times, I don't think its worth to clutter it with another function